### ISSUE TITLE
configs of 2d and 3d augment

### ISSUE THREAD

**Issue by wzmsltw** [mit-han-lab/bevfusion#39]
Dear authors, thanks for your nice work! I tried to finetune the pretrained detection model with some modification (using lr 1e-4 10 epochs without any data augmentation). Although the training loss is very low, the detection mAP in val set is bad (65 mAP on epoch 5 and 63 mAP on epoch10). It seems without data augmentation, the overfitting is severe. Can you provide the setting of **augment2d and augment3d**? I do not need other settings for now. Thanks so much!
